### ISSUE TITLE
View a collection accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3962]
### Descriptive summary

In testing for the Hyrax 3.0 release, the following points are actions that need to be taken on the Hyrax collection view page in order to be more accessible:

**Elements must have sufficient color contrast:**
![Screen Shot 2019-09-11 at 4 14 17 PM](https://user-images.githubusercontent.com/9621070/64741938-3a0ff380-d4af-11e9-8a95-8287d2646790.png)

![Screen Shot 2019-09-11 at 4 14 54 PM](https://user-images.githubusercontent.com/9621070/64741953-54e26800-d4af-11e9-80ae-ffe1f3c762fd.png)

![Screen Shot 2019-09-11 at 4 15 32 PM](https://user-images.githubusercontent.com/9621070/64741970-67f53800-d4af-11e9-9fd2-7857ac341fae.png)

![Screen Shot 2019-09-11 at 4 15 57 PM](https://user-images.githubusercontent.com/9621070/64741992-75aabd80-d4af-11e9-9c6a-4974149c1019.png)

**Form elements must have labels:**
![Screen Shot 2019-09-11 at 4 16 41 PM](https://user-images.githubusercontent.com/9621070/64742019-907d3200-d4af-11e9-81b7-94dfa2aed94b.png)

**ARIA role must be appropriate for the element:**
![Screen Shot 2019-09-11 at 4 18 52 PM](https://user-images.githubusercontent.com/9621070/64742100-ddf99f00-d4af-11e9-98a0-11d82386d6b1.png)

**Headings must not be empty:**
![Screen Shot 2019-09-11 at 4 19 32 PM](https://user-images.githubusercontent.com/9621070/64742126-f669b980-d4af-11e9-9f37-2e55f04ff7ab.png)

**Heading levels should only increase by one:**
![Screen Shot 2019-09-11 at 4 20 04 PM](https://user-images.githubusercontent.com/9621070/64742142-097c8980-d4b0-11e9-8400-64fd0c1854f5.png)

**Banner landmark must not be contained in another landmark:**
![Screen Shot 2019-09-11 at 4 20 40 PM](https://user-images.githubusercontent.com/9621070/64742171-21540d80-d4b0-11e9-921f-3036c4cdc7df.png)

**Document must not have more than one banner landmark:**
![Screen Shot 2019-09-11 at 4 21 24 PM](https://user-images.githubusercontent.com/9621070/64742197-3892fb00-d4b0-11e9-9ec6-23e64f46442a.png)

**Ensures landmarks are unique:**
![Screen Shot 2019-09-11 at 4 21 54 PM](https://user-images.githubusercontent.com/9621070/64742218-4b0d3480-d4b0-11e9-9d5a-a7db7a39bf93.png)

![Screen Shot 2019-09-11 at 4 22 18 PM](https://user-images.githubusercontent.com/9621070/64742237-595b5080-d4b0-11e9-88bd-469ab525ceca.png)

**All page content must be contained by landmarks:**
![Screen Shot 2019-09-11 at 4 22 54 PM](https://user-images.githubusercontent.com/9621070/64742261-7132d480-d4b0-11e9-9f86-9c83027442e1.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/collections/td96k2517?locale=en
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, unable to tab through all the links such as the work links. 

Works well in Chrome. 

![Screen Recording 2019-09-16 at 01 46 PM](https://user-images.githubusercontent.com/9621070/64992341-8fba1680-d888-11e9-8b67-5ae18a2a423a.gif)

**Comment by dlim87** [comment]
# tab test
Working on firefox, chrome, and safari

# Axe audit 
13 issues:
## Images must have alternative text (1)
![Image 2019-11-06 at 10 22 25 AM](https://user-images.githubusercontent.com/5492162/68325963-5d09ee80-007f-11ea-8177-ad10430a1c47.png)
## links must have discernible text (1)
![Image 2019-11-06 at 10 23 28 AM](https://user-images.githubusercontent.com/5492162/68326039-862a7f00-007f-11ea-8568-a4131e9bea44.png)
## Headings must not be empty (1)
![Image 2019-11-06 at 10 24 07 AM](https://user-images.githubusercontent.com/5492162/68326076-9a6e7c00-007f-11ea-8f8a-8ec810ab4fc4.png)
## Heading level should only increase by one (1)
![Image 2019-11-06 at 10 24 35 AM](https://user-images.githubusercontent.com/5492162/68326124-b1ad6980-007f-11ea-86d1-c18dd313f71f.png)
## Banner landmark must not be contained by another landmark (1)
![Image 2019-11-06 at 10 25 24 AM](https://user-images.githubusercontent.com/5492162/68326183-cee23800-007f-11ea-9b0e-41396725d2c0.png)
## Document must not have more than one banner landmark (1) 
<img width="582" alt="Image 2019-11-06 at 10 27 12 AM" src="https://user-images.githubusercontent.com/5492162/68326349-1b2d7800-0080-11ea-87ad-da1c99faec54.png">
## Ensure landmarks are unique (1)
<img width="777" alt="Image 2019-11-06 at 10 28 43 AM" src="https://user-images.githubusercontent.com/5492162/68326491-65aef480-0080-11ea-8a76-b67f5ffaa6b7.png">
## All page content must be contained by landmarks (1)
<img width="584" alt="Image 2019-11-06 at 10 29 08 AM" src="https://user-images.githubusercontent.com/5492162/68326527-752e3d80-0080-11ea-88c4-db491804b29f.png">
## Elements must have sufficient color contrast (5)
<img width="336" alt="Image 2019-11-06 at 10 31 13 AM" src="https://user-images.githubusercontent.com/5492162/68326653-b292cb00-0080-11ea-9bec-0bc08f9b2e0d.png">
<img width="578" alt="Image 2019-11-06 at 10 32 38 AM" src="https://user-images.githubusercontent.com/5492162/68326784-f4237600-0080-11ea-9315-ac51b216a85c.png">
<img width="584" alt="Image 2019-11-06 at 10 32 48 AM" src="https://user-images.githubusercontent.com/5492162/68326798-f8e82a00-0080-11ea-930e-ed9a20b6a36b.png">
<img width="587" alt="Image 2019-11-06 at 10 33 28 AM" src="https://user-images.githubusercontent.com/5492162/68326810-fd144780-0080-11ea-885e-efb3901ec511.png">
<img width="581" alt="Image 2019-11-06 at 10 33 37 AM" src="https://user-images.githubusercontent.com/5492162/68326822-01406500-0081-11ea-9776-d1e5e89de0aa.png">
<img width="424" alt="Image 2019-11-06 at 10 31 36 AM" src="https://user-images.githubusercontent.com/5492162/68326830-056c8280-0081-11ea-8eae-71f154898e45.png">
<img width="575" alt="Image 2019-11-06 at 10 33 44 AM" src="https://user-images.githubusercontent.com/5492162/68326838-09000980-0081-11ea-8488-93198ceccc10.png">

**Comment by dlim87** [comment]
This was fixed. The only complaint it continued to have was the color contrast in the footer (the very last one in the above post), which is an implementation detail specific to hyrax
